train.py

```diff
@@ -1186,6 +1240,16 @@ def main(
     group: str = None, # For wandb logging
     entity: str = None, # For wandb logging
     n_bits: int = 4, # passed to hqq
-    profiling_output: str = "", # Output file prefix for profiling
+    profile: bool_arg = False, # Whether to profile with torch.profiler
+    profiling_output: str = "profiles", # Output file prefix for profiling
+    with_stack: bool_arg = False, # Output stacks for profiling. Note that setting export_memory_timeline will automatically export traces since `with_stack` must be true to profile memory.
```

Comment:
If we weren't refactoring, it would probably be good to make profiler-specific args eaisly identifiable (eg with a prof_ prefix). For now we can hold off until the refactoring to address this question of how to organize the arg list though.

Reply:
Ok if we leave as is?  All the profiling args are demarcated from rest of `CLI` args and are off by default.  Also, documentation clearly lays out the meaning of each of these args.

Reply:
Yes fine to leave as is.